net/upnp/Makefile

```
@@ -2,7 +2,7 @@ PLUGIN_NAME=		upnp
 PLUGIN_VERSION=		1.5
 PLUGIN_REVISION=	5
```

```suggestion
PLUGIN_REVISION=	6
```

Since we are making changes we need to let the package manager know :)